### ISSUE TITLE
Fields Optimization 

### ISSUE THREAD

**Issue by 2amarkina** [blockframes/blockframes#6949]
Need to see if possible to optimize the length of fields according to the data inside. 
That could help us to visually simplify the long forms such as Rights Details cards or Movie Tunnel.

**Comment by Jackseed** [comment]
Would you have an example of an input that could benefit this optimization? I'm not sure to get it right

**Comment by 2amarkina** [comment]
Let's take this part for example. 
We know the exact max number of characters used in the select with format. It's length could be cut. Same for the Running Time field that takes all the line. It could be much shorter. 
![Снимок экрана 2021-11-19 в 13 59 00](https://user-images.githubusercontent.com/55499036/142626964-422f00e5-5691-47fe-9a1f-1c543bc74d2a.png)

For the moment, it's a low priority idea and I guess that it can take quite a lot time on the dev side.

**Comment by Jackseed** [comment]
@GrandSchtroumpf could you have a look to this proposition and give an estimation on the amount of changes it would request?

**Comment by GrandSchtroumpf** [comment]
With a size fixed by @stephanie-cascade8 we could apply a grid or something like that. Maybe half a day of work to implement.
We cannot have dynamic size depending on the content.

**Comment by stephanie-cascade8** [comment]
> With a size fixed by @stephanie-cascade8 we could apply a grid or something like that. Maybe half a day of work to implement. We cannot have dynamic size depending on the content.

I have a little inventory of fields that need a fixed size (eg: date fields, yes/no boolean select ... stuff like that) I think 3 different sizes would be enough. Let's make it a UI issue if you want @2amarkina

**Comment by frenchyrules** [comment]
W.I.P [here](https://www.figma.com/file/X18CCQHVcWOtGfVetF4FKs/Fields-optimization?node-id=6%3A479)

**Comment by cams-goussale** [comment]
@2amarkina @GrandSchtroumpf we just discussed this issue with @frenchyrules @dmoncadalv 

We're not super convinced with the idea of having 3 sizes for the fields. We're afraid it would be just like the modale: by standarizing all the fields to 3 sizes, we break some places in the app where it's currently working well.

The only place where we've identified a real need for fields optimization is the movie form. And we'll have a dedicated issue / discussion for it specifically.

@2amarkina have you identified other places where there is a need to optimize fields?
